src/ui/public/agg_types/controls/top.html

```diff
@@ -0,0 +1,29 @@
+<div class="form-group">
```

Comment:
Since this template is only for the sort input, could we give the file a more specific name like `top_sort.html`?